protocols/peer/contracts/Peer.sol

```diff
@@ -84,7 +143,7 @@ contract Peer is IPeer, Ownable {
     uint256 _maxTakerAmount,
     uint256 _priceCoef,
     uint256 _priceExp
-  ) external onlyOwner {
+  ) external onlyAdmins {
```

Comment:
The `only*` syntax is to enforce the _role_ of the caller, so feel `onlyAdmin` would be most correct.